tailwind.config.js

```diff
@@ -1,9 +1,9 @@
 /** @type {import('tailwindcss').Config} */
 module.exports = {
   content: [
-    './sanity/**/*.{js,ts,jsx,tsx}',
-    './pages/**/*.{js,ts,jsx,tsx}',
-    './components/**/*.{js,ts,jsx,tsx}',
```

Comment:
reason for this change?

Reply:
Yes, my rational for removing the tsx from each of these was that it seemed the easiest way to prevent tailwind from impacting our components/pages styled with MUI. Since most of the previous iteration was written in JSX, the styling should remain on the existing site, while we build out the new site in Typescript.